tslint.json

```diff
@@ -1,5 +1,6 @@
 {
-    "extends": [
-        "@dcos/tslint-config"
-    ]
+  "extends": ["@dcos/tslint-config"],
+  "rules": {
+    "interface-name": [false]
```

Comment:
Ah, I see, we disabled the "I". Can we then be consistent and not use it anywhere?

Reply:
We could, but I think this should be the scope of a Jira for changing it in the global config. If you insist I will add it nonetheless to get this one moved on faster.

Reply:
I agree. I think the only thing that is really needed here is to now steamroll the previous team decision. I am in favor of this being here, and the global one happened in another ticket.